packages/post-rector/src/Rector/NameImportingPostRector.php

```diff
@@ -85,6 +85,10 @@ private function processNodeName(Name $name): ?Node
     {
         $importName = $this->getName($name);
 
+        if ($name->isSpecialClassName()) {
```

Comment:
this `if` seems can be moved up before `$importName` definition.

Reply:
Fixed!